WordPress/Classes/Models/Blog.m

```diff
@@ -64,8 +68,8 @@ - (void)didTurnIntoFault
     [super didTurnIntoFault];
 
     // Clean up instance variables
-    _blavatarUrl = nil;
-    _api = nil;
+    self.blavatarUrl = nil;
+    self.api = nil;
```

Comment:
This _might_ have side effects, since `_api` has a lazy getter. Can we swich it back, just in case?. Reason is:

- The Blog instance might be loaded in multiple ManagedObjectContext's, associated to multiple threads.
- Each time a Blog instance, in any thread, gets faulted, a *WPXMLRPCClient* instance would be created, and for dotcom scenarios, this _might_ mean that the keychain would be accessed.

Odds are, it's safe to hit the getter (it'd just have an extra overhead). But playing it safe!.